package-manifest.toml

```diff
@@ -123,7 +123,7 @@ source.type = "local"
 source.rust.binary_names = ["oximeter", "clickhouse-schema-updater"]
 source.rust.release = true
 source.paths = [
```

Comment:
(No action needed, just trying to grok this) - I thought you were trying to figure out how to bundle the network zone into each zone that needs it, but I don't see that as a dependency of this package anywhere. 

I see that the call to `zone_network_setup_install` adds the SMF-level dependency, but where are we actually shoving the "rust network service binary" into this zone?

Reply:
oooof... good catch! :grimacing: 

Why are the "helios / deploy" tests passing though? Shouldn't the zone crap out because of the SMF-level dependency?

Reply:
Practically speaking, `oximeter` being unavailable doesn't preclude launching an instance. Propolis was reworked some time ago to continue attempting to register as a producer in the background.

Reply:
Yup! This is all working as intended,  TL;DR is "don't trust the helios/deploy job to tell you that Oximeter is up".